code/modules/surgery/tools.dm

```diff
@@ -192,6 +212,11 @@
 	origin_tech = "biotech=1"
 	attack_verb = list("slapped")
 
+/obj/item/surgical_drapes/suicide_act(mob/living/user)
+ 	user.visible_message("<span class='suicide'>[user] drapes [src] over their head and communicates to the spooky ghost god! It looks like [user.p_theyre()] trying to ascend to the next plane!</span>")
+ 	dust(mob/user)
+	return(BRUTELOSS) //if they somehow don't get dusted i guess

```

Comment:
this is probably going to derp up something since it's giving bruteloss to a now null object.

Reply:
https://github.com/tgstation/tgstation/blob/551ede6d35bc955fa83979049441321f46ab9653/code/__HELPERS/pronouns.dm#L67

it does.